source/connect.txt

```diff
@@ -123,3 +141,88 @@ For details, see :ref:`comp-command-line-connect`.
    Favorite Connections </connect/favorite-connections>
    Required Access </connect/required-access>
    Disconnect </connect/disconnect>
+
+Connection String Examples
+--------------------------
+
+The following examples highlight some advanced connection string options.
+For information on all connection string options, see :ref:`connections-connection-options`.  
+
+TLS Options
+~~~~~~~~~~~
+
+The following connection string enables TLS using the ``tls`` URI option. 
+``tlsCertificateKeyFile`` specifies the path to the client certificate and 
+private key. Specifying the ``tlsCertificateKeyFile`` in your connection string
+is optional, unless the server requires client certificates, or if 
+you are using X.509 authentication.
+
+.. code-block:: 
+
+   mongodb://myDatabaseUser:D1fficultP%40ssw0rd@db0.example.com/?directConnection=true&tls=true&tlsCAFile=path/to/rootCA.crt&tlsCertificateKeyFile=path/to/server_certificate.pem
+
+To enable SSL, use the equivalent ``ssl=true`` option in place of ``tls=true``. 
+
+X.509 Authentication
+````````````````````
+
+The following example demonstrates how to specify X.509 authentication in your connection string
+using the :urioption:`authMechanism` connection string option:
+
+.. code-block:: 
+
+   mongodb://myDatabaseUser:D1fficultP%40ssw0rd@db1.example.com/?authMechanism=MONGODB-X509&tls=true&tlsCertificateKeyFile=%2Fpath%2Fto%2Fclient_certificate.pem
```

Comment:
[nit]

I think we should update `&tlsCertificateKeyFile=%2Fpath%2Fto%2Fclient_certificate.pem` to use slashes like:

```
&tlsCertificateKeyFile=/path/to/client_certificate.pem
```